src/features/VisualNovel/pages/VisualNovelPage/VisualNovelPage.tsx

```diff
@@ -56,7 +56,7 @@ export const VisualNovelPage: FC = () => {
             </div>
             <VisualNovelTabs id={id} />
           </header>
-          <div className={cls.tabContent}>
+          <div>
```

Comment:
Extra `div`? If not, let's use `Box` instead

Reply:
I don't think we should mix two styling approaches here. This component styled with css, and adding `Box` would lead to inconsistency and confusion.